src/devtools/domains/history.js

```diff
@@ -33,12 +36,24 @@ class History {
       }
     }
 
-    return { ...newHistory, list }
+    return { ...newHistory, list, focused }
+  }
+
+  focus(_old, id) {
```

Comment:
an underscore is used to denote a variable that won't be used (both to future developers and often to linters/compilers), since you're using the previous state here, i'd just go with `old`